app/helpers/job_invocations_helper.rb

```diff
@@ -33,7 +33,7 @@ def preview_hosts(template_invocation)
   end
 
   def collapsed_preview(target)
-    title = target || 'N/A'
+    title = target.to_s || 'N/A'
```

Comment:
`nil.to` returns an empty string so `||` no longer works. Should we use
```suggestion
    title = (target || 'N/A').to_s
```